### PR TITLE
Replace props.intl.formatMessage with <FormattedMessage />

### PR DESCRIPTION
**Pull request by gabrielrab**
Hi!
I am sending my PR to resolve [Issue #20](https://github.com/meedan/check/issues/20)

### CODE REVIEW

File: src/app/components/media/CreateRelatedMediaDialog.js [hunk 1]
```diff
@@ -97,7 +97,7 @@ class CreateRelatedMediaDialog extends React.Component {
         </DialogContent>
         <DialogActions>
           <Button id="create-media-dialog__dismiss-button" onClick={this.props.onDismiss}>
-            {this.props.intl.formatMessage(globalStrings.cancel)}
+            <FormattedMessage {...globalStrings.cancel}/>
```
Comment (amoedoamorim) [inline src/app/components/media/CreateRelatedMediaDialog.js:100]: Please insert a space after the closing curly brace `} />`

File: src/app/components/task/EditTaskDialog.js [hunk 2]
```diff
@@ -238,13 +230,13 @@ class EditTaskDialog extends React.Component {
         ))}
         <div style={{ marginTop: units(1) }}>
           <Button onClick={this.handleAddValue.bind(this)}>
-            {this.props.intl.formatMessage(messages.addValue)}
+            <FormattedMessage id='singleChoiceTask.addValue' defaultMessage='Add Option'/>
```
Comment (amoedoamorim) [inline src/app/components/task/EditTaskDialog.js:233]: Please use double quotes for strings

File: src/app/components/task/EditTaskDialog.js [hunk 3]
```diff
@@ -238,13 +230,13 @@ class EditTaskDialog extends React.Component {
...
           </Button>
           <Button
             onClick={this.handleAddOther.bind(this)}
             disabled={this.state.hasOther}
           >
-            {this.props.intl.formatMessage(messages.addOther)}
+            <FormattedMessage id='singleChoiceTask.addOther'  defaultMessage='Add "Other"'  />
```
Comment (amoedoamorim) [inline src/app/components/task/EditTaskDialog.js:239]: Please use double quotes for `id` string. Fine to leave `defaultMessage` as is

**Review by amoedoamorim — CHANGES_REQUESTED** [review]
Thanks you @gabrielrab for your contribution! We appreciate it! In order for it to be accepted and merged please apply the requested changes.

**Comment by gabrielrab** [comment]
@amoedoamorim 
I complete change requests

File: src/app/components/task/EditTaskDialog.js [hunk 4]
```diff
@@ -238,13 +230,13 @@ class EditTaskDialog extends React.Component {
...
           <Button onClick={this.handleAddValue.bind(this)}>
-            {this.props.intl.formatMessage(messages.addValue)}
+            <FormattedMessage id="singleChoiceTask.addValue" defaultMessage="Add Option"/>
           </Button>
           <Button
             onClick={this.handleAddOther.bind(this)}
             disabled={this.state.hasOther}
           >
-            {this.props.intl.formatMessage(messages.addOther)}
+            <FormattedMessage id="singleChoiceTask.addOther"  defaultMessage='Add "Other"'  />
```
Comment (amoedoamorim) [inline src/app/components/task/EditTaskDialog.js:239]: 239:63  error  Multiple spaces found before 'defaultMessage'  no-multi-spaces
239:93  error  Multiple spaces found before '/'               no-multi-spaces

**Review by amoedoamorim — CHANGES_REQUESTED** [review]
Sorry, can you please also fix these? Have you tried building the app locally? Again, thanks for contributing!

**Comment by gabrielrab** [comment]
@amoedoamorim 
I can run only check-web or i need run all check stack ?

**Comment by amoedoamorim** [comment]
> I can run only check-web or i need run all check stack ?
Hi @gabrielrab , sorry for taking so long to reply. Are you still around? While you theoretically could do it, you'd need to manually install whatever dependencies are needed. The supported way to run Check is to build and run the whole stack.

**Review by amoedoamorim — CHANGES_REQUESTED** [review]
Hey @gabrielrab ! Thanks for contributing! We appreciate it! Please apply the requested changes so we can accept and merge your PR!

**Comment by amoedoamorim** [comment]
Hello @gabrielrab , are you still around? Please let me know if you're still interested in fixing this!

**Comment by gabrielrab** [comment]
@amoedoamorim yes, I'm here! I am also interested in solving this problem.

**Comment by amoedoamorim** [comment]
> @amoedoamorim yes, I'm here! I am also interested in solving this problem.

That's nice! There are some simple changes requested, please fix them before someone else does!

**Comment by gabrielrab** [comment]
@amoedoamorim before 45 minutes of the second half, I solved the problem. Sorry for the delay !

**Pull request merged**